### PR TITLE
Add mac to functions Quickstart

### PR DESCRIPTION
**Pull request by granluo**

### CODE REVIEW

File: functions/FunctionsExample/Shared/View/CapitalizeMessageView.swift [hunk 1]
```diff
@@ -20,6 +20,11 @@ import Firebase
 struct CapitalizeMessageView: View {
   @State private var comment: String = ""
   @State private var outcome: String = ""
+  #if os(iOS)
+  let roundRetangleFillColor = Color(.systemGray5)
```
Comment (morganchen12) [inline functions/FunctionsExample/Shared/View/CapitalizeMessageView.swift:24]: nit: `roundRetangleFillColor` -> `roundRectangleFillColor`

File: functions/FunctionsExample/Shared/View/AddNumbersView.swift [hunk 2]
```diff
@@ -21,6 +21,13 @@ struct AddNumbersView: View {
   @State private var num1: String = ""
   @State private var num2: String = ""
   @State private var outcome: String = ""
+  #if os(iOS)
+  let textForegroundColor = Color(UIColor.secondaryLabel)
+  let roundRetangleFillColor = Color(.systemGray5)
```
Comment (morganchen12) [inline functions/FunctionsExample/Shared/View/AddNumbersView.swift:26]: nit: `roundRetangleFillColor` -> `roundRectangleFillColor`

**Review by morganchen12 — APPROVED** [review]
LGTM, thanks for deduplicating the views :)

**Comment by granluo** [comment]
Oops, just accidentally merged, @ncooke3 please feel free to review and leave any comments. Will update with another PR.

**Pull request merged**